modules/ROOT/pages/migrate-mule3-apimanager.adoc

```diff
@@ -7,19 +7,19 @@ You migrate your existing Mule 3 applications that are managed by API Manager to
 
 . <<prepare-migration,Prepare for your migration>>
 +
-Based on how your existing Mule 3 application is configured, carefully review the prerequisites and determine how to plan the migration.
+Based on how your existing Mule 3 application is configured, carefully review the prerequisites to prepare for the migration.
 . <<determine-migration-approach,Determine the migration approach>>
 +
-Based on several factors, such as whether the migration requires a downtime, requires removing applied policies, or requires migrating contracts, you use the existing API instance or create an instance for migration.
+Based on several factors, such as whether the migration requires a downtime, requires removing applied policies, or requires migrating contracts, you use an existing API instance or create an instance for the migration.
 . <<perform-migration,Perform the migration>>
 +
-Based on whether you want to create an API or use an existing one, you can appropriately migrate your application.
+Migrate your application using an existing or a new API.
 
 
 [[prepare-migration]]
 == Prepare to Migrate Your Mule 3 Applications
 
-Before you migrate your existing Mule 3 applications that are managed by API Manager to Mule 4, carefully review the following guidelines and determine:
+Before you migrate your existing Mule 3 applications managed by API Manager to Mule 4, carefully review the following guidelines and determine:
```

Comment:
Another option:
```suggestion
Before you migrate your existing API-managed Mule 3 applications to Mule 4, carefully review the following guidelines and determine:
```

Reply:
Is it necessary to qualify the applications as managed by API Manager?
If so, need we continue to repeat this after establishing it (as in Line 37 below)?